admin-console/src/main/webapp/src/components/EmailTemplates/RenderEmailTemplate.tsx

```diff
@@ -83,14 +83,13 @@ const RenderedTemplate: React.FunctionComponent<RenderedTemplateProps> = props =
 };
 
 export const RenderEmailTemplateForm: React.FunctionComponent<EmailTemplateFormProps> = props => {
-    const emailTemplate = useRenderEmailRequest();
+    const emailTemplate = useRenderTemplateRequest();
```

Comment:
Creating a separate service to render only template instead of the subject / body. That's why we were having an extra empty string